Era 937 put back daml hub instructions (#139)
digital-asset/ex-healthcare-claims-processing: README.md

### Stopping the App

> 1. Stop every running command by pressing **Ctrl+C**.

Author: ```suggestion
1. Stop each running command by pressing **Ctrl+C**.
```

> 1. Stop the every running command by pressing **Ctrl+C**.

Author: ```suggestion
1. Stop every running command by pressing **Ctrl+C**.
```

> 1. Stop the every running command by pressing **Ctrl+C**.

Author: ```suggestion
1. Stop each running command by pressing **Ctrl+C**.
```

## Working with Daml Hub

> **Note**: the custom UI does not work on Daml Hub yet, only the builtin Live Explorer.

Reviewer: ```suggestion
**Note**: The custom UI does not work on Daml Hub yet, but you can use Daml Hub's live explorer to interact with the contracts by following the instructions below.
```
